CHANGES.md

```diff
@@ -1,6 +1,48 @@
 Release Changes List
 ====================
 
+version 1.24.1
+==============
+
+Update to the twenty-seventh public release of Chapel, April 2021  
+(see also changes below for 1.24.0)
+
+Highlights
+----------
+* significant performance improvements for InfiniBand systems
+* improved support for computing with `enum` ranges
+* closed all known memory leaks
+
+Feature Improvements
+--------------------
+* extended `param` for-loops to support `enum` ranges
+* added support for open-interval `enum` ranges
+
+Performance Optimizations / Improvements
+----------------------------------------
+* improved NUMA affinity and startup times when using a fixed heap
```

Comment:
I was wondering if we should call out that the gasnet upgrade had large performance improvements in this section.

Reply:
Would you like to propose some text?  (I'm not clear on when it does/does not apply)

Reply:
I was just thinking something like :

```
* improved performance on InfiniBand systems by upgrading GASNet versions"
```

or 

```
* improved performance of GASNet over `ibv` segment `large` 
```

or 

```
* improved performance of applications that communicate a significant amount of memory on InfiniBand systems
```

but this last one is too long. It's basically under gasnet-ibv-large if a node does communication of more than 20k memory regions (which are 128KB) then we saw performance regressions before. This occurs for single serial transfers that are larger than 3GB, but also impacts other applications that are communicating a non-trivial number of memory regions like ISx/NPB-FT or Arkouda at scale or arkouda when running a bunch of benchmarks. I'm struggling with how to put this succinctly though in a way that makes sense to end users. Maybe just 

```
* improved communication performance on InfiniBand systems
```

to keep things simple?